src/perform-bundle/index.ts

```diff
@@ -0,0 +1,53 @@
+// perform-bundle/index.ts
+
+import process from 'node:process';
+import { readFile } from 'node:fs/promises';
+import { debug } from 'debug';
+import type { Metafile } from 'esbuild';
+
+import { publishCommentAndRemovePrevious } from '../github-api';
+import runDeployer from './deployer';
+import analyze from './analyze';
+
+const log = debug('perform-bundle');
+
+function bytesToKB(bytes: number) {
+  return Math.round(bytes / 1024);
+}
+
+async function readMetaDataFile(): Promise<Metafile> {
+  try {
+    const rawFile = await readFile('esbuild-lambda/metafile.json', 'utf8');
+    return JSON.parse(rawFile) as Metafile;
+  } catch (error) {
+    log('Exception thrown attempting to read meta data file: ', JSON.stringify(error));
+    throw error;
+  }
+}
+
+export async function main(): Promise<void> {
+  log('Action start');
+  await runDeployer();
+  const metaDataFile = await readMetaDataFile();
```

Comment:
assuming runDeployer is ultimately executing `builder`, these same stats should already be printed to stdout so could we use that instead of re-calculating?

Reply:
I'll need to make some changes to deployer to support that. The output of builder isn't returned by deployer at the moment.

Reply:
yeah now I think about it, the better solution would be to have typescript-config export the `builder` and `analyze` functions, so you can use that directly.  i created an issue for another day: https://github.com/checkdigit/typescript-config/issues/50

Reply:
Yeah that would be the simplest solution. I can go have a go at doing that and then come back to this.